pkg/logicalbackup/logicalbackup.go

```diff
@@ -58,8 +58,8 @@ type LogicalBackup struct {
 	replMessageWaitTimeout time.Duration
 	statusTimeout          time.Duration
 
-	relations     map[message.Identifier]message.Relation
```

Comment:
`Identifier` is a too generic name for the fields it stores, as for `NamespacedName` looks much better ([hi, k8s!](https://github.com/kubernetes/apimachinery/blob/ed135c5b96450fd24e5e981c708114fbbd950697/pkg/types/namespacedname.go#L31-L34))